src/SFML/Window/Unix/GlxContext.cpp

```diff
@@ -269,9 +272,6 @@ void GlxContext::display()
 ////////////////////////////////////////////////////////////
 void GlxContext::setVerticalSyncEnabled(bool enabled)
 {
-    // Make sure that extensions are initialized
-    ensureExtensionsInit(m_display, DefaultScreen(m_display));
```

Comment:
These calls used to be present in more places right before extensions were used. As code got moved around, the ensuring and using of the extensions moved further apart and we ended up in the current state. Logically it really only makes sense to load the extensions in the constructors and static methods, ensuring that they are always loaded whenever we do anything after.